sources/light_client.move

```diff
@@ -106,16 +107,16 @@ public fun new_light_client(params: Params, start_height: u64, trusted_headers:
 ///
 /// Header serialization reference:
 /// https://developer.bitcoin.org/reference/block_chain.html#block-headers
-public fun init_light_client(params: Params, start_height: u64, trusted_headers: vector<vector<u8>>, start_chain_work: u256, ctx: &mut TxContext) {
+public fun init_light_client(params: Params, start_height: u64, trusted_headers: vector<vector<u8>>, parent_chain_work: u256, ctx: &mut TxContext) {
```

Comment:
please document the `parent_chain_work` and what it should be.

Reply:
done!